pkgs/development/libraries/hare-ev/default.nix

```
@@ -0,0 +1,40 @@
+{ stdenv
+, lib
+, fetchFromSourcehut
+, hare
+, unstableGitUpdater
+}:
+
+stdenv.mkDerivation rec {
+  pname = "hare-ev";
+  version = "unstable-2023-07-29";
+
+  src = fetchFromSourcehut {
+    owner = "~sircmpwn";
+    repo = "hare-ev";
+    rev = "680e7e93732401e310cbcf597e4a8c1d6079e607";
+    hash = "sha256-miXIdA++Q7GEZNyBv3zuWx20BZPB/3p3heBW0DWKdoI=";
+  };
+
+  nativeCheckInputs = [
+    hare
+  ];
+
+  preCheck = ''
+    export HARECACHE=$(mktemp -d)
+  '';
+
+  installFlags = [ "PREFIX=$(out)" ];
+
+  doCheck = true;
+
+  passthru.updateScript = unstableGitUpdater { };
+
+  meta = with lib; {
+    description = "an event loop for Hare programs";
```

```suggestion
    description = "Event loop for Hare programs";
```
`meta.description` should start with a capital letter.